aggregator/list.go

```diff
@@ -37,6 +38,10 @@ import (
 	"github.com/uber-go/tally"
 )
 
+const (
+	defaultNumPartitions = 1024
```

Comment:
nit: move this to Options?

Reply:
This is mostly just an optimization to allocate some initial capacity, doubt we'd ever need to change it so probably not worth it to make it optional.